project3/index.html

```diff
@@ -0,0 +1,410 @@
+            <input id="real-name" type="text" required>
+
+            <label for="alias">Alias<input id="alias" type="text" required>
+            </label>
+
+
+            <label for="email">Email <input id="email" type="text">
+            </label>
+
+            <h3 class="personal">personal</h3>
+            <h3 class="characteristics">characteristics</h3>
+
+            <label for="age">Age<input id="age" type="number" required></label>
+
+            <fieldset>
+                <div class="categories">
+                    <legend>Category</legend>
+                    <p>Please select as many as apply to your origin and how you gained your powers.</p>
+
+                    <span class=" category">
+                        <input name="categroy" id="alien" type="checkbox">
+                        <label for="alen">Alien</label></span>
+
+                    <span class="category">
+                        <input name="category" id="cyborg" type="checkbox">
+                        <label for="cyborg">Cyborg</label></span>
+
+                    <span class="category">
+                        <input name="category" id="experiment" type="checkbox">
+                        <label for="experiment">Experiment</label></span>
+
+                    <span class="category">
+                        <input name="category" id="human" type="checkbox">
+                        <label for="human">Human</label></span>
+
+                    <span class="category">
+                        <input name="category" id="inhuman" type="checkbox">
+                        <label for="inhuman">Inhuman</label></span>
+
+                    <span class="category">
+                        <input name="category" id="mutant" type="checkbox">
+                        <label for="mutant">Mutant</label></span>
+                </div>
+            </fieldset>
+
+            <fieldset>
+                <div class="gender">
+                    <legend>Gender</legend>
+
+                    <span class="category">
+                        <input name="gender" id="female" type="radio" required>
+                        <label for="female">Female</label></span>
+
+                    <span class="category">
+                        <input name="gender" id="male" type="radio" required>
+                        <label for="male">Male</label></span>
+
+                    <span class="category">
+                        <input name="gender" id="nonbinary" type="radio" required>
+                        <label for="nonbinary">Nonbinary</label></span>
+
+                    <span class="category">
+                        <input name="gender" id="genderqueer" type="radio" required>
+                        <label for="genderqueer">Genderqueer/Genderfluid</label></span>
+
+                    <span class="category">
+                        <input name="gender" id="answer" type="radio" required>
+                        <label for="answer">Prefer not to answer/label></span>
```

Comment:
This element isn't properly closed:
```suggestion
                        <label for="answer">Prefer not to answer</label></span>
```